photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinate.scala

```diff
@@ -190,7 +192,12 @@ object RandomEffectCoordinate {
     //TODO: Need a better design that properly unpersists the broadcasted variables and persists the computed RDD
     val modelsForPassiveDataBroadcast = passiveData.sparkContext.broadcast(modelsForPassiveData)
     val passiveScores = passiveData.mapValues { case (randomEffectId, labeledPoint) =>
-      modelsForPassiveDataBroadcast.value(randomEffectId).computeScore(labeledPoint.features)
+      ScoredGameDatum(
```

Comment:
A ```KeyValueScore.apply``` is also needed - I needed one when I wrote the snippet above for `FixedEffectCoordinate.score`.